app/src/main/java/com/example/makeiteven2/fragments/FragmentArcadeModeScreen.kt

```diff
@@ -445,7 +455,7 @@ class FragmentArcadeModeScreen : Fragment(), View.OnClickListener, IFinishTimer
                 } else {
                     //you loose
                     Handler().postDelayed({
-                        showFinishDialog(Constants.LOSE_DIALOG)
+                        //showFinishDialog(Constants.LOSE_DIALOG)
```

Comment:
also?

Reply:
I'm not sure if to show dialog here or not, see after merge it's in your fragments responsibility